homeworkk8.1/homeworkk8.1/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace homework8._1
+{
+    class Program
+    {
+        static char horizontalLine = (char)0x2500;
+        static char topLeftCorner = (char)0x250C;
+        static char topRightCorner = (char)0x2510;
+        static char bottomLeftCorner = (char)0x2514;
+        static char bottomRightCorner = (char)0x2518;
+        static char verticalLeft = (char)0x2524;
+        static char verticalRight = (char)0x251C;
+        static char horizontalDown = (char)0x252C;
+        static char horizontalUp = (char)0x2534;
+        static char verticalHorizontal = (char)0x253C;
```

Comment:
Было бы красивее использовать вертикальную черту для рисования рамок
```suggestion
        static char verticalHorizontal = (char)0x253C;
        static char vertical = (char)0x2502;
```